boefjes/tests/test_sucurity_txt.py

```diff
@@ -0,0 +1,90 @@
+            "primary_key='URL|internet|https://www.example.nl/.well-known/security.txt', "
+            "network=Reference('Network|internet'), "
+            "raw=AnyUrl('https://www.example.nl/.well-known/security.txt', "
+            "scheme='https', host='www.example.nl', tld='nl', host_type='domain', "
+            "path='/.well-known/security.txt'), web_url=None), Hostname(object_type='Hostname', "
+            "scan_profile=None, primary_key='Hostname|internet|www.example.nl', "
+            "network=Reference('Network|internet'), name='www.example.nl', dns_zone=None, "
+            "registered_domain=None), IPAddressV4(object_type='IPAddressV4', scan_profile=None, "
+            "primary_key='IPAddressV4|internet|1.8.8.8', address=IPv4Address('1.8.8.8'), "
+            "network=Reference('Network|internet'), netblock=None), Service(object_type='Service', "
+            "scan_profile=None, primary_key='Service|https', name='https'), IPPort(object_type='IPPort', "
+            "scan_profile=None, primary_key='IPPort|internet|1.8.8.8|tcp|443', "
+            "address=Reference('IPAddressV4|internet|1.8.8.8'), protocol=<Protocol.TCP: 'tcp'>, port=443, "
+            "state=None), IPService(object_type='IPService', scan_profile=None, "
+            "primary_key='IPService|internet|1.8.8.8|tcp|443|https', "
+            "ip_port=Reference('IPPort|internet|1.8.8.8|tcp|443'), service=Reference('Service|https')), "
+            "Website(object_type='Website', scan_profile=None, "
+            "primary_key='Website|internet|1.8.8.8|tcp|443|https|internet|www.example.nl', "
+            "ip_service=Reference('IPService|internet|1.8.8.8|tcp|443|https'), "
+            "hostname=Reference('Hostname|internet|www.example.nl'), certificate=None), "
+            "SecurityTXT(object_type='SecurityTXT', scan_profile=None, "
+            "primary_key='SecurityTXT|internet|1.8.8.8|tcp|443|https|internet|www.example.nl|internet|"
+            "https://www.example.nl/.well-known/security.txt', "
+            "website=Reference('Website|internet|1.8.8.8|tcp|443|https|internet|www.example.nl'), "
+            "url=Reference('URL|internet|https://www.example.nl/.well-known/security.txt'), "
+            "redirects_to=None, security_txt='This is the content'), SecurityTXT(object_type='SecurityTXT', "
+            "scan_profile=None, primary_key='SecurityTXT|internet|8.8.8.8|tcp|443|https|internet|example.nl|"
+            "internet|https://example.nl/.well-known/security.txt', "
+            "website=Reference('Website|internet|8.8.8.8|tcp|443|https|internet|example.nl'), "
+            "url=Reference('URL|internet|https://example.nl/.well-known/security.txt'), "
+            "redirects_to=Reference('SecurityTXT|internet|1.8.8.8|tcp|443|https|internet|www.example.nl|"
+            "internet|https://www.example.nl/.well-known/security.txt'), security_txt=None)]"
```

Comment:
This is very hard to read, verify or modify. Why not create proper objects to the assertion?

It would also be better to use example.com, example.net or example.com because they are defined in RFC 6761. For IP addresses it would be best to use the one of the ranges from https://datatracker.ietf.org/doc/html/rfc5737#section-3